File: examples/petstore/oseg_generated/csharp/PlaceOrder_default_example.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPITools.Api;
using Org.OpenAPITools.Client;
using Org.OpenAPITools.Model;

public class PlaceOrderDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        var order = new Order(
            id: 12345,
            petId: 98765,
            quantity: 5,
            shipDate: DateTime.Parse("2025-01-01T17:32:28Z"),
            status: Order.StatusEnum.Approved,
            complete: false
        );

        try
        {
            var response = new StoreApi(config).PlaceOrder(
                order: order
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Store#PlaceOrder: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: examples/petstore/oseg_generated/php/PlaceOrderDefaultExample.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$config = OpenAPI\Client\Configuration::getDefaultConfiguration();

$order = (new OpenAPI\Client\Model\Order())
    ->setId(12345)
    ->setPetId(98765)
    ->setQuantity(5)
    ->setShipDate(new \DateTime("2025-01-01T17:32:28Z"))
    ->setStatus(OpenAPI\Client\Model\Order::STATUS_APPROVED)
    ->setComplete(false);

try {
    $response = (new OpenAPI\Client\Api\StoreApi(config: $config))->placeOrder(
        order: $order,
    );

    print_r($response);
} catch (OpenAPI\Client\ApiException $e) {
    echo 'Exception when calling Store#placeOrder: '
        . print_r($e->getResponseObject());
}
